sh_scrapy/log.py

```diff
@@ -52,7 +52,7 @@ def initialize_logging():
         pass
 
     nh = logging.NullHandler()
-    for ln in ('boto', 'requests', 'hubstorage'):
+    for ln in ('boto', 'requests', 'scrapinghub'):
```

Comment:
I think you should keep this line untouched or update loggers in python-scrapinghub 
https://github.com/scrapinghub/python-scrapinghub/blob/master/scrapinghub/hubstorage/batchuploader.py#L18